business/istio_config.go

```diff
@@ -1126,16 +1187,16 @@ func (in *IstioConfigService) CreateIstioConfigDetail(namespace, resourceType st
 	return istioConfigDetail, err
 }
 
-func (in *IstioConfigService) IsGatewayAPI() bool {
-	return in.k8s.IsGatewayAPI()
+func (in *IstioConfigService) IsGatewayAPI(cluster string) bool {
+	return in.userClients[cluster].IsGatewayAPI()
 }
 
 // Check if istio Ambient profile was enabled
 // ATM it is defined in the istio-cni-config configmap
 func (in *IstioConfigService) IsAmbientEnabled() bool {
```

Comment:
Should this take `cluster` as an arg as well?

Reply:
I think so. But, the only thing to take into account is that we should have access to the Istio Control plane - We won't have one in a primary-remote